k6/tests/retryFailedJobsOnStartupDuringQueueProcessing.js

```diff
@@ -84,6 +84,4 @@ export default function () {
       return r.status == 200;
     },
   });
-
-  sleep(1);
```

Comment:
What there a point to this sleep? @jannisvisser

Reply:
don't know, but probably not.